src/components/views/Home.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchLoadedData } from '../redux/loadApi';
+import styles from './Home.module.css';
+
+const continent = [
+  {
+    id: 1,
+    route: 'africa',
+    name: 'Africa',
+    numberOfCountries: 58,
+  },
+  {
+    id: 2,
+    route: 'asia',
+    name: 'Asia',
+    numberOfCountries: 50,
+  },
+  {
+    id: 3,
+    route: 'europe',
+    name: 'Europe',
+    numberOfCountries: 48,
+  },
+  {
+    id: 4,
+    route: 'north-america',
+    name: 'North America',
+    numberOfCountries: 39,
+  },
+  {
+    id: 5,
+    route: 'south-america',
+    name: 'South America',
+    numberOfCountries: 14,
+  },
+  {
+    id: 6,
+    route: 'australia-oceania',
+    name: 'Australia Oceania',
+    numberOfCountries: 19,
+  },
+];
+
+let isFetching = true;
+
+function Home() {
+  const dispatch = useDispatch();
+
+  if (isFetching === true) {
+    isFetching = false;
+    dispatch(fetchLoadedData());
+  }
+
+  return (
+    <div className={styles.homePage}>
+      <div>
+        <h1 className={styles.homeHeader}>
+          The World
+          <br />
+          COVID-19
+        </h1>
+        <p>Statistics</p>
+      </div>
+      <ul className={styles.continents}>
+        {continent.map((item) => (
+          <li key={item.id} className={styles.continent}>
+            <div className={`box ${item.route}`}>
+              <div className="overlay">
+                <Link to={`/continent/${item.route}`} exact="true" className="dataPage">
+                  <h2>{item.name}</h2>
+                  <p>{`${item.numberOfCountries} Countries`}</p>
+                </Link>
+              </div>
+            </div>
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+}
+
```

Comment:
- [ ] Good job with your project. I really appreciate the effort you made to implement these features. So, from the project's requirement, I noticed that you should add another feature where you use a **Filter stateless component** to filter data retrieved from the API. i.e As it is about countries I suggest you add a **search bar** where a user can type for example a name of a country he wants to get information from. I know you can do that :muscle: